.github/workflows/check_content.yml

```diff
@@ -0,0 +1,22 @@
+name: check_content
+
+on:
+  # Intentionally check on all branches, to detect any issues as early as possible
```

Comment:
Do we really want to check this on all branches, or only on PRs? What if we have "non-working code" that we need to push because others want to work on it?

Reply:
In this case, "non-working code" would be markdown content that is broken...

I'm curious to understand what about having these checks turned on would prevent someone from pushing code in a non-working state?

Reply:
Ah. Brainfart on my side that this isn't much of a coding project. Non-issue!